modules/setting/mailer.go

```diff
@@ -71,7 +71,9 @@ func newMailService() {
 		givenHost := sec.Key("HOST").String()
 		addr, port, err := net.SplitHostPort(givenHost)
 		if err != nil {
-			log.Fatal("Invalid mailer.HOST (%s): %v", givenHost, err)
+			if err.Error() != "missing port in address" {
+				log.Fatal("Invalid mailer.HOST (%s): %v", givenHost, err)
```

Comment:
Seeing as `mailer.HOST` is deprecated, shouldn't this log something like `mailer.SMTP_ADDR`?

Reply:
But error is about host setting being invalid, this will be removed when we drop HOST support

Reply:
Gotcha